### PR TITLE
Implements #185

### PR DESCRIPTION
**Pull request by kldavis4**

### CODE REVIEW

File: html/admin/index.html [hunk 1]
```diff
@@ -110,13 +113,27 @@ <h1>Admin Password</h1>
         </form>
     </div>
 
+    <div class="container" id="banner" style="display: none;">
+      <div class="page-header">
+        <h1>User Interface</h1>
+      </div>
+      <p class="lead">Banner Message</p>
```
Comment (edwinsteele) [inline html/admin/index.html]: As this takes HTML, let’s make that clear by changing this to “Banner Message (html or plain text)"

**Comment by kldavis4** [comment]
<script> tags are already filtered out by react, but you can add javascript event handlers (like onclick) to html elements in the banner, which we probably don't want to allow. I'm not sure how to filter that out in nginx, though.

Can you provide an example banner string that triggers that console error? I tried adding inline styles to an anchor tag but didn't get any errors.

**Comment by edwinsteele** [comment]
This triggers it for me: ```<div style="color: red;">something</div>```
I’m using it in Safari and I get both the css.map and js.map error. In firefox I just get the js.map error, but I get it regardless of whether I put CSS in the banner. I don’t see either in Chrome.

**Comment by kldavis4** [comment]
Thanks.. that's interesting that it only happens in chrome and safari

**Comment by kldavis4** [comment]
Turns out the error was due to an issue with how the icon ui build works - looks like chrome just wasn't displaying the error - and it wasn't related to the banner stuff

. I fixed it here: https://github.com/ConnectBox/connectbox-react-icon-client/commit/44ca9a6a721a054903b10cc6ef1859868e1ae424

**Pull request merged**